.testing/Makefile

```diff
@@ -700,31 +720,28 @@ work/unit/std.out: build/unit/MOM_unit_tests
 	    cat p2.std.err | tail -n 100 ; \
 	  )
 
-build/unit/codecov:
-	mkdir -p $(@D)
-	cd $(@D) \
-	  && curl -s $(CODECOV_UPLOADER_URL) -o $(@F)
-	chmod +x $@
-
-# Use driver coverage file as a proxy for the run
+# NOTE: .gcov actually depends on .gcda, but .gcda is produced with std.out
 # TODO: Replace work/unit/std.out with *.gcda?
-build/unit/MOM_new_unit_tests.gcov: work/unit/std.out
-	mkdir -p $(@D)
+build/unit/MOM_file_parser_tests.F90.gcov: work/unit/std.out
```

Comment:
Shouldn't this target be called `build/unit/gcov.unit.out:`, since that is what is produced? The dependency for `report.cov.unit:` would need to be changed too.

Reply:
I guess... they are both outputs of the rule.  The advantage of the `.gcov` file is that it is actually used in the subsequent rule (`report.cov.unit`) whereas `gcov.unit.out` is just the stdout of the command and would just be a proxy.

Admittedly, `MOM_file_parser_tests.F90.gcov` is a proxy for all of the `.gcov` files, but using this feels more correct than a text output file unused by the subsequent rule.

Reply:
As you know, I'm about to add another unit test. The current pattern would suggest the source file being tested to also be listed here. Because this .gcov file is a proxy for all the others, this will work as we add tests but then the question is why this source file only?

Reply:
I guess I just had to pick one, so I chose the one which was actually tested.

It doesn't need to be this file.  And should probably be "all" of the files.  But if you think a better one would work then we can select that one.  Or we can come up with some way that considers all of them.

But you are the Makefile guru, so if you see a better solution then I would trust your judgement.

Reply:
FYI this comment is in the file :D
```
# NOTE: Using file parser gcov report as a proxy for test completion
```

Reply:
I think we can come back to this. I'd rather see this go live, and until we have other unit tests along side it's hard to testwhere the weak points are